graph-manager-docs/source/managed-federation/setup.mdx

```diff
@@ -0,0 +1,134 @@
+
+## 1. Register your implementing services
+
+To enable managed federation, each of your data graph's [implementing services](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) must register its schema with Graph Manager:
+
+```mermaid
+graph BT;
+  subgraph " "
+  gateway([Gateway]);
+  serviceA[Products service];
+  serviceB[Reviews service];
+  serviceC[Inventory service];
+  end
+  graphManager{{Apollo Graph Manager}};
+  gateway --- serviceA & serviceB & serviceC;
+  serviceA & serviceB & serviceC -. apollo service:push .-> graphManager;
+  class graphManager secondary;
+```
+
+To get set up with schema registration, do the following for each of your implementing services:
+
+1. Complete the following prerequisites:
+
+  <ProjectConfigPanel />
+
+2. In the service's `apollo.config.js` file, add `name` and `url` fields to the `service` object:
+
+    ```json{3-4}:title=apollo.config.js
+    module.exports = {
+      service: {
+        name: "Products"
+        url: "http://products-graphql.svc.cluster.local:4001/"
+        endpoint: {
+          url: "http://localhost:4000"
+        }
+      }
+    };
+    ```
+
+    * The `name` field uniquely identifies the service within your data graph. Its value appears throughout the Graph Manager UI.
+    * The `url` field points to your service's GraphQL endpoint. To secure your data graph, this should be a URL that can be reached by your [gateway](https://www.apollographql.com/docs/apollo-server/federation/gateway/), but _cannot_ be reached via the public internet.
+
+3. From your project's root directory, run the following:
+
+    ```
+    apollo service:push --tag=staging
+    ```
+
+    The CLI obtains all necessary configuration from your project's `.env` and `apollo.config.js` files, then pushes your schema to Graph Manager under the `staging` variant.
+
+> When you're ready to register your services in your production environment, you can either remove the `--tag` option to push to the default variant (named `current`) or replace `staging` with a new variant (such as `production`).
+
+## 2. Configure continuous delivery
+
+With managed federation, Graph Manager (_not_ your gateway) is responsible for combining your individual schemas into a single federated schema. Consequently, **each service in your data graph must re-register its schema every time the schema changes.** Otherwise, your gateway can't obtain the latest federated schema from Graph Manager.
+
+The best way to accomplish this is to add the `apollo service:push` call to your continuous delivery pipeline. Every time you push a schema change to production, this call should automatically accompany that push.
+
+For more information on best practices for updating your services, see [Pushing configuration updates safely](./advanced-topics/#pushing-configuration-updates-safely).
```

Comment:
Do you think it's worth a quick note on recommending that all schema changes are made in a backwards compatible way (e.g. adding fields, only removing or modifying deprecated fields that are used) or is it obvious?

Reply:
Adding a line to the "Pushing configuration updates safely" section